From klaas.van.gend@mvista.com  Tue Oct 27 10:34:48 2009
From: Klaas van Gend <klaas.van.gend@mvista.com>
Date: Thu, 22 Oct 2009 07:51:00 +0200
Subject: staging: comedi: multiq3: add KERN_
To: gregkh@suse.de
Cc: Klaas van Gend <klaas.van.gend@mvista.com>
Message-ID: <1256190660-14913-2-git-send-email-klaas.van.gend@mvista.com>


Add KERN_ to printk statements to reduce the number of warnings
shown by checkpatch.pl.

Signed-off-by: Klaas van Gend <klaas.van.gend@mvista.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/multiq3.c |   11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

--- a/drivers/staging/comedi/drivers/multiq3.c
+++ b/drivers/staging/comedi/drivers/multiq3.c
@@ -252,9 +252,9 @@ static int multiq3_attach(struct comedi_
 	struct comedi_subdevice *s;
 
 	iobase = it->options[0];
-	printk("comedi%d: multiq3: 0x%04lx ", dev->minor, iobase);
+	printk(KERN_INFO "comedi%d: multiq3: 0x%04lx ", dev->minor, iobase);
 	if (!request_region(iobase, MULTIQ3_SIZE, "multiq3")) {
-		printk("comedi%d: I/O port conflict\n", dev->minor);
+		printk(KERN_ERR "comedi%d: I/O port conflict\n", dev->minor);
 		return -EIO;
 	}
 
@@ -262,9 +262,10 @@ static int multiq3_attach(struct comedi_
 
 	irq = it->options[1];
 	if (irq)
-		printk("comedi%d: irq = %u ignored\n", dev->minor, irq);
+		printk(KERN_WARNING "comedi%d: irq = %u ignored\n",
+			dev->minor, irq);
 	else
-		printk("comedi%d: no irq\n", dev->minor);
+		printk(KERN_WARNING "comedi%d: no irq\n", dev->minor);
 	dev->board_name = "multiq3";
 	result = alloc_subdevices(dev, 5);
 	if (result < 0)
@@ -328,7 +329,7 @@ static int multiq3_attach(struct comedi_
 
 static int multiq3_detach(struct comedi_device *dev)
 {
-	printk("comedi%d: multiq3: remove\n", dev->minor);
+	printk(KERN_INFO "comedi%d: multiq3: remove\n", dev->minor);
 
 	if (dev->iobase)
 		release_region(dev->iobase, MULTIQ3_SIZE);
